src/screens/Home/EventList/EventList.tsx

```diff
@@ -23,22 +25,53 @@ interface EventListProps extends EventListPropsFromRedux {
 const EventList: React.FC<EventListProps> = ({
   setCurrentEventId,
   setCurrentInvitationId,
+  currentUser,
   search,
 }) => {
   const navigation = useNavigation();
   const { loading, error, data } = useQuery<GetFoodDrives>(GET_FOOD_DRIVES, {
     pollInterval: config.defaultPollInterval,
   });
+
+  const {
+    loading: loadingInvitation,
+    error: invitationError,
+    data: invitationData,
+  } = useQuery<GetUserInvitation>(GET_INVITATIONS_BY_USER, {
+    pollInterval: config.defaultPollInterval,
+    variables: {
+      getInvitationsUserId: currentUser?.id,
+    },
+  });
+
   const toastProvider = useToastProvider();
   const styles = useStyles();
 
+  const currentUserRegisteredEvents = invitationData?.getInvitations.map(
+    (invitation) => invitation.event.id,
+  );
+  //get food drives excluding current user's registered events
+  const foodDrives = useMemo(
+    () =>
+      data?.getFoodDrives.filter((foodDrive) => {
```

Comment:
This approach works, but kind of defeats the purpose of GraphQL's ability to support FE queries declaratively. For this ticket, we could update the BE to take in a variable that specifies that we want to exclude events that the user is invited to and then modify the `GET_FOOD_DRIVES` query to take in that variable.

Also, having two live queries might affect performance of the app.

Reply:
I'll put this in another ticket

Reply:
I'll cancel this PR and do that ^